source/deployment/cluster.rst

```diff
@@ -420,6 +420,15 @@ Does Mattermost support multi-region High Availability deployment?
 
 Yes. Although not officially tested, you can set up a cluster across AWS regions, for example, and it should work without issues.
 
+What does Mattermost recommend for diaster recovery of the databases?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When deploying Mattermost in a High Availability configuration, we recommend using a database load balancer between Mattermost and your database. Depending on your deployment this needs more or less consideration.
+
+For example, if deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple availability zones.
+
+When deploying Mattermost on your own cluster please refer to your IT team for a solution best suited for your existing architecture.
+
```

Comment:
3/5: The two sentences in one paragraph with a minor rewording:
"For example, if you're deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple Availability Zones. If you're deploying Mattermost on your own cluster please consult with your IT team for a solution best suited for your existing architecture."